rdata.c

```diff
@@ -805,6 +806,21 @@ rdata_svcparam_alpn_to_string(buffer_type *output, uint16_t val_len,
 	return 1;
 }
 
+static int
+rdata_svcparam_tls_supported_groups_to_string(buffer_type *output,
+		uint16_t val_len, uint16_t *data)
+{
+	assert(val_len > 0); /* Guaranteed by rdata_svcparam_to_string */
+
+	if ((val_len % sizeof(uint16_t)) == 0)
```

Comment:
This seems to fail multiples of uint16_t, perhaps it should check for not a multiple of uint16_t.

```suggestion
	if ((val_len % sizeof(uint16_t)) == 1)
```

Reply:
Good catch! Thanks!

Reply:
It wasn't picked up by the `svcb.tdir` test because both the presentation format and the wire format zone files translated to RFC3597 unknown type format. The two translations were then compared.
I have now manually checked that it now does print in presentation format.